app/views/layouts/oaisys/application.builder

```diff
@@ -1,7 +1,28 @@
 xml.instruct! :xml, version: '1.0'
-xml.tag!('OAI-PMH', 'xmlns': 'http://www.openarchives.org/OAI/2.0/',
-                    'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
-                    'xsi:schemaLocation': 'http://www.openarchives.org/OAI/2.0/
-                     http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd') do
+xml.tag!('OAI-PMH',
+         if @short_header
+           { 'xmlns': 'http://www.openarchives.org/OAI/2.0/',
+             'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
+             'xsi:schemaLocation': 'http://www.openarchives.org/OAI/2.0/
+              http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd' }
+         else
+           { 'xmlns:dc': 'http://purl.org/dc/elements/1.1/',
```

Comment:
You can get rid of the whole `@short_header` variable and this `if/else`. Part of what I'm working on is to capture these namespaces for the tags that are in them, so that we can dynamically inject these namespaces as-needed only when the response contains tags that are in the namespace.

How we'll really want to do this is something like `content_for` coupled with a yield here, to allow the serialization gem the opportunity to push additional headers in. See: https://api.rubyonrails.org/classes/ActionView/Helpers/CaptureHelper.html

Reply:
(even if we can't get it coming in through the gem, we could potentially just add these headers statically in the views for the responses that normally would have them, via the `content_for` mechanism)